### PR TITLE
WW-95 go to login page if anon editing disabled tests added

### PR DESCRIPTION
**Pull request by idradm**
Please do not merge before https://github.com/Wikia/app/pull/11352 is on production

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class SpecialCommunityPageTests extends NewTestTemplate {
+
+  @Execute(onWikia = "muppet", asUser = User.ANONYMOUS, mockAds = "true")
+  public void verifyAnonEditingForbiddenFlow() {
+    SpecialCommunity page = new SpecialCommunity();
+    // click the first link from any card
+    WebElement link = page.getLinkFromCards();
+    String url = getEditUrl(link.getText(), "veaction=edit");
+    link.click();
+    // should be at register page
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: How do we check that? It's not clear

File: src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    String url = getEditUrl(link.getText(), "action=edit");
+
+    link.click();
+
+    Assertion.assertEquals(url, page.getCurrentUrl());
+  }
+
+  private String getEditUrl(String page, String query) {
+    return urlBuilder.appendQueryStringToURL(
+        urlBuilder.getUrlForPage(String.format("/wiki/%s", page.replace(" ", "_"))),
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: What do you think about moving the normalization logic to 
getUrlForPage(String pageName) ?

File: src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java (hunk 1 above)
Comment (idradm) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: We do not. The test should fail if it's not true.

Comment (lukaszkonieczny) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: +1 Can we verify we are on register page. Also do we want to cover case when new user is joining community?

Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: Are you sure? loginAs from WikiBasePageObject uses Helios to log in, what is the connection with register / login page ?

Comment (idradm) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java]: I will add assert for it to make sure.

**Pull request merged**